### PR TITLE
New keys added to mfc default

### PR DESCRIPTION
**Pull request by orensbruli**

### CODE REVIEW

**Comment by iceman1001** [comment]
Interesting,
Do you know if the keys are definite static,   or are they diversified per card?

**Comment by iceman1001** [comment]
Did you manage to check with another card?

**Comment by orensbruli** [comment]
Sorry. Forgot. Checking...

**Comment by iceman1001** [comment]
Good, good. 
Now you just need to resolve the conflict...
Then I can merge it.

File: client/dictionaries/mfc_default_keys.dic [hunk 1]
```diff
@@ -399,38 +403,142 @@ A160FCD5EC4C
...
+C9BE49675FE4
+CCCE24102003
+CDE668FDCDBA
+CFE63749080A
+cfe63749080a
+D23A31A4AAB9
+D61707FFDFB1
+d61707ffdfb1
+DD0DE3BA08A6
+dd0de3ba08a6
```
Comment (iceman1001) [inline client/dictionaries/mfc_default_keys.dic:531]: This part seems to contains several duplicates.

**Comment by iceman1001** [comment]
I notice your patch contains multiple duplicates.   That will not do.

**Comment by orensbruli** [comment]
> I notice your patch contains multiple duplicates. That will not do.

Yes, I saw and that's why I put this again as a draft. I think I have fixed it, could you check again? Thx.

**Comment by iceman1001** [comment]
much work for a simple PR...

**Comment by orensbruli** [comment]
> much work for a simple PR...

I think it's easier now. It's passing the unique keys workflow.

**Comment by iceman1001** [comment]
I have to give it to you,     adding the uniq workflow is a great move.

How about you remove the draft on this PR so I can finally merge it?

**Comment by iceman1001** [comment]
Thank you!

**Pull request merged**